pages/en/lb3/Attaching-models-to-data-sources.md

```diff
@@ -132,6 +149,14 @@ you can specify the data source you want it to use from among those you've added
 [Data source generator](Data-source-generator.html) and the default `db`
 data source (that uses the [memory connector](Memory-connector.html)).
 
+```shell
+? Enter the model name: myModel
```

Comment:
Might wanna add how to create model..  to run `slc loopback:model` or `apic create --type model` for API Connect (similar format as what you have in this file)